.github/workflows/ci-backend-cql.yml

```diff
@@ -87,6 +87,9 @@ jobs:
           - module: cql
             args: "-Pcassandra3-murmur-client-auth -Dtest=\"**/diskstorage/cql/CQLStoreTest.java\""
             name: murmur-client-auth
+          - module: cql
+            args: "-Pscylladb -Dcassandra.docker.image=scylladb/scylla -Dcassandra.docker.version=4.3.0 -Dtest.cql.excludes=MEMORY_TESTS,PERFORMANCE_TESTS,BRITTLE_TESTS"
```

Comment:
Aren't "MEMORY_TESTS,PERFORMANCE_TESTS,BRITTLE_TESTS" already included in "<test.excluded.groups>"?

Reply:
I removed them in the latest commit.